front/pages/w/[wId]/a/[aId]/settings.tsx

```diff
@@ -85,6 +85,7 @@ export default function SettingsView({
     if (appName.length == 0) {
       setAppNameError("");
       return false;
+      // eslint-disable-next-line no-useless-escape
```

Comment:
nit: should we remove the useless escape ?

Reply:
Same here probably just `\_` but need to be tested